src/emailAlert.ts

```diff
@@ -0,0 +1,35 @@
+const sendgrid = require('@sendgrid/mail')
+
+import { error, log } from './debug'
+
+const api_key = process.env.SENDGRID_API_KEY
+const alert_email = process.env.ALERT_EMAIL
+
+export const sendEmailAlert = (message: string) => {
+    if (!api_key || !alert_email) {
+        log('Email Alerts not configured, not sending email alert.')
+        return
+    }
+
+    sendgrid.setApiKey(api_key)
+
+    const msg = {
+        to: alert_email,
+        from: 'mokhtar@jsgenesis.com',
```

Comment:
I should probably make this configurable as well..

Reply:
Makes sense :+1: